src/Keboola/StorageApi/Client.php

```diff
@@ -171,11 +167,14 @@ private function initClient()
         ]);
 
         $handlerStack->push((RequestTimeoutMiddleware::factory()));
-        $handlerStack->push(Middleware::log(
+
+        // this needs to run after http_errors resolves so the error has proper log level
+        $handlerStack->before('http_errors', Middleware::log(
```

Comment:
v commentu je after, v kodu before

Reply:
no to je chytak, sice se to registruje `before`, ale vykona se to `after` :smile: Predpokaldam, ze je to proto, ze je to handler **stack** a pri requestu se vyhodnoti middleware v poradi (log before http_errors) a pri response zase v opacnym poradi.